.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/ for more details
+#
+
+references:
+  working_directory: /src
+  defaults: &defaults
+    docker:
+      - image: golang:1.11-alpine
+
+  install_sw_requirements: &install_sw_requirements
+    run:
+      name: install_sw_requirements
+      command: |
+        apk add ca-certificates git openssh gcc libc-dev
+
+  get_dependencies: &get_dependencies
+    run:
+      name: get-dependencies
+      command: |
+        go get github.com/teamdigitale/anpr-dashboard-server/server
+        go get github.com/teamdigitale/anpr-dashboard-server/converter
+        go get github.com/stretchr/testify/assert
```

Comment:
I would replace it with a more comprehensive `go get -d ./...`

Reply:
Unfortunately, this starts to recursively download a bunch of dependencies from all over and the operation lasts "forever". I'd suspect it's not only downloading the dependencies needed

Reply:
we will investigate in a separate issue.